### PR TITLE
array mapping programs-square of numbers and displaying output as str…

### PR DESCRIPTION
**Pull request by SimiS22**
Please review the code.
1. Output of array is square of numbers in the array.
2. Convert the numbers in a array to string.

### CODE REVIEW

File: src/squared-numbers/index.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+export function squaredNumbersUsingForLoop(input: number[]) {
+
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty lines

File: src/squared-numbers/index.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
+export function squaredNumbersUsingForLoop(input: number[]) {
+
+
+    const result: number[] = [];
+    for (let i = 0; i < input.length; i++) {
+        result.push(Math.pow(input[i], 2));
+
+    }
+    return result;
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty lines

File: src/squared-numbers/index.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        result.push(Math.pow(input[i], 2));
+
+    }
+    return result;
+
+}
+
+const square = (value: number) => {
+    return Math.pow(value, 2)
+};
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: add a empty line after a function block

File: src/squared-numbers/index.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    return input.map(square);
+}
+
+export const arrayMappingUsingArrowFunction = (input: number[]) => {
+    const output: string[] = [];
+    for (let i = 0; i < input.length; i++) {
+        switch (input[i]) {
+            case 0:
+                output[i] = 'zero';
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty line

File: src/squared-numbers/index.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+                output[i] = 'nine';
+
+                break;
+
+            default:
+                output[i] = 'not 0 to 9';
+                break;
+        }
+
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty lines

File: src/squared-numbers/index.ts [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    }
+    console.log(output);
+    return output;
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty line

File: src/squared-numbers/index.ts [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+
```
Comment (chin2km) [inline src/squared-numbers/index.ts]: remove unnecessary empty line

File: src/squared-numbers/squared-numbers.test.ts [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+describe('square of numbers in an array', function () {
+    it('should return square of numbers in an array', function () {
+        expect(squaredNumbersUsingForLoop([1, 2, 3, 4, 5])).toEqual([1, 4, 9, 16, 25]);
+    })
+
+    it('should return square of numbers in an array', function () {
+        expect(squaredNumbersUsingMapFunction([1, 2, 3, 4, 5])).toEqual([1, 4, 9, 16, 25]);
+    })
+
+
```
Comment (chin2km) [inline src/squared-numbers/squared-numbers.test.ts]: remove unnecessary empty line

**Comment by SimiS22** [comment]
As suggested,removed the unnecessary empty lines.

**Comment by chin2km** [comment]
> As suggested,removed the unnecessary empty lines.

but I cant see the changes here!

**Pull request merged**